controller/src/test/java/io/pravega/controller/server/v1/ControllerServiceTest.java

```diff
@@ -260,6 +260,26 @@ public void testTransactions() {
         Controller.TxnStatus status = consumer.commitTransaction(SCOPE, stream1, txnId, "", 0L).join();
         assertEquals(status.getStatus(), Controller.TxnStatus.Status.FAILURE);
 
+        status = consumer.abortTransaction(SCOPE, stream1, txnId).join();
+        assertEquals(status.getStatus(), Controller.TxnStatus.Status.FAILURE);
+
+        doThrow(StoreException.create(StoreException.Type.DATA_NOT_FOUND, "Data Not Found"))
+                .when(streamStore).sealTransaction(eq(SCOPE), eq(stream1), eq(txnId), anyBoolean(), any(), anyString(), anyLong(),
+                any(), any());
+
+        status = consumer.commitTransaction(SCOPE, stream1, txnId, "", 0L).join();
+        assertEquals(status.getStatus(), Controller.TxnStatus.Status.FAILURE);
+
+        status = consumer.abortTransaction(SCOPE, stream1, txnId).join();
+        assertEquals(status.getStatus(), Controller.TxnStatus.Status.FAILURE);
+
+        doThrow(StoreException.create(StoreException.Type.ILLEGAL_STATE, "Data Not Found"))
+                .when(streamStore).sealTransaction(eq(SCOPE), eq(stream1), eq(txnId), anyBoolean(), any(), anyString(), anyLong(),
+                any(), any());
+
+        status = consumer.commitTransaction(SCOPE, stream1, txnId, "", 0L).join();
+        assertEquals(status.getStatus(), Controller.TxnStatus.Status.FAILURE);
+
```

Comment:
can you please add another test covering the case where an arbitrary exception is thrown by the store resulting in that same exception being thrown from the api

Reply:
Throwing of an unknown exception is tested on line 256. Just above the cases I have added.  This is for seal transaction, but since commit and abort utilize that pathway, I assumed the case is covered. Is it not?